hivemind/moe/server/server.py

```diff
@@ -313,14 +302,14 @@ def shutdown(self):
 
 
 @contextmanager
-def background_server(*args, shutdown_timeout=5, **kwargs) -> Tuple[Endpoint, List[Multiaddr]]:
-    """A context manager that creates server in a background process, awaits .ready on entry and shuts down on exit"""
+def background_server(*args, shutdown_timeout=5, **kwargs) -> PeerInfo:
+    """A context manager that creates server in a background thread, awaits .ready on entry and shutdowns on exit"""
```

Comment:
```suggestion
    """A context manager that creates server in a background thread, awaits .ready on entry and shuts down on exit"""
```

nit: `shutdown` is a noun, `shut down` is a verb